content/500-platform/10-about.mdx

```diff
@@ -42,6 +42,33 @@ In each workspace, you can:
 - manage billing, i.e. select a [subscription plan](https://www.prisma.io/pricing?utm_source=docs&utm_medium=platform-docs), configure payment methods, or view the invoice history.
 - view the usage of your enabled PDP products across all projects in that workspace.
 - invite other users to collaborate in the workspace.
+- access the [Optimize](/optimize) dashboard to measure query performance and receive AI-powered recommendations.
```

Comment:
is this linking to the optimize page in the docs? do you mean for it to go to the dashboard?